### PR TITLE
Add item description

### PR DESCRIPTION
**Pull request by BenCheung0422**
This reserves the feature of item description, which might be useful when there are colorable items and more items based on non-name-indicated attributes. Taken from #324.

### CODE REVIEW

File: src/main/java/minicraft/screen/PlayerInvDisplay.java [hunk 1]
```diff
@@ -7,54 +7,77 @@
...
+import minicraft.screen.entry.StringEntry;
 
 public class PlayerInvDisplay extends Display {
 
 	private static final int padding = 10;
 
 	private final Player player;
 
+	private String itemDescription = "";
+	private Menu.Builder descriptionMenuBuilder;
```
Comment (Makkkkus) [inline src/main/java/minicraft/screen/PlayerInvDisplay.java:25]: You don't need this.

**Review by Makkkkus — CHANGES_REQUESTED** [review]

File: src/main/java/minicraft/screen/PlayerInvDisplay.java (hunk 1 above)
Comment (BenCheung0422) [reply src/main/java/minicraft/screen/PlayerInvDisplay.java:25]: Why we don't need it? It is a dynamic menu size.

Comment (Makkkkus) [reply src/main/java/minicraft/screen/PlayerInvDisplay.java]: Just don't keep the builder stored.

Comment (BenCheung0422) [reply src/main/java/minicraft/screen/PlayerInvDisplay.java]: The builder is always stored every time the frame sizes are modified. It also appears in other places as commonly.

**Review by Makkkkus — APPROVED** [review]

**Review by ChristofferHolmesland — APPROVED** [review]

**Pull request merged**